stores/GlobalPath/GlobalPathReducers.ts

```diff
@@ -0,0 +1,27 @@
+import BaseReducer from "@/utils/BaseReducer";
+import GlobalPathActions from "./GlobalPathActions";
+import { GlobalPathState } from "./GlobalPathTypes";
+import { AnyAction } from "redux";
+
+export default class GlobalPathReducer extends BaseReducer {
+    initialState: GlobalPathState = {
+        data: {
+            waypoints: [{latitude: 49.37614179786771, longitude: -123.27376619978901}]
```

Comment:
Let's initialize `waypoints` to an empty array. We have this initialized for GPS because we can predetermine the starting point for the sailboat, but we don't know the predetermined path for globalpath yet